src/components/TabularReport/styles.css

```diff
@@ -0,0 +1,33 @@
+.tabular-report table {
+    border-collapse: collapse;
+}
+
+.tabular-report table thead tr {
```

Comment:
It's not too important in this case, but having multiple parts in a selector slows down rendering performance.
No need to change that here, I guess. But good to keep it in mind that BEM will def. be more performant than this.